### PR TITLE
DietPi-Software | PHP: Fix 'upload_tmp_dir' issue

### PR DESCRIPTION
**Pull request by MichaIng**
- https://github.com/Fourdee/DietPi/issues/1144
+ reorder installation to use own ini file.

### CODE REVIEW

File: dietpi/dietpi-software [hunk 1]
```diff
@@ -8832,15 +8832,18 @@ _EOF_
 			# - Apache2 has its own PHP module
 			if (( ${aSOFTWARE_INSTALL_STATE[83]} >= 1 )); then
 
-				echo -e "nothing here" &> /dev/null
+				# https://github.com/Fourdee/DietPi/issues/1144
+				local php_service=/lib/systemd/system/apache2.service
```
Comment (Fourdee) [inline dietpi/dietpi-software]: Please define as characters/string eg:
```local php_service='/lib/systemd/system/apache2.service'```

Prevents any leakage to bash.

Comment (MichaIng) [reply dietpi/dietpi-software]: Yes can do this here. But for `/lib/systemd/system/php*-fpm.service` it does not work, as `*` is then not interpreted as optional `5` or `7.0` dependant on PHP version. Tried with `'` and `"`...
Tested on Jessie and Stretch, and without string definition it works fine. `bash -x testscript` looks like it is in the end interpreted as string but replaces `*` correctly.

Do you think it is okay to leave it as path for second statement, or might there be other problems with it? Of course I can also just do `$DISTRO` check and then define it as fixed string.

Comment (Fourdee) [reply dietpi/dietpi-software]: @MichaIng 

```
root@DietPi:~# test='/root/test*'
root@DietPi:~# echo 1 > test3
root@DietPi:~# echo 1 > test2
root@DietPi:~# echo 1 > test1
root@DietPi:~# ls -lha | grep test
-rw-r--r--  1 root root    2 Nov 27 17:43 test1
-rw-r--r--  1 root root    2 Nov 27 17:43 test2
-rw-r--r--  1 root root    2 Nov 27 17:43 test3
root@DietPi:~# rm "$test"
rm: cannot remove '/root/test*': No such file or directory
root@DietPi:~# rm $test
root@DietPi:~# ls -lha | grep test
root@DietPi:~#
```

```
php_service='/lib/systemd/system/php*-fpm.service'
```
Should work using:
```
 $php_service
```

Comment (MichaIng) [reply dietpi/dietpi-software]: @Fourdee 
Ah yes, I tested with `test='~/test*'` and there it is not working, seems to be not interpreted as path with possible variables `*` anymore. But beginning with `/root` instead works, so will also work in this script. Adjustment done

**Comment by Fourdee** [comment]
@MichaIng 

Legend 👍.

**Pull request merged**